bookApp/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+class keywords(models.Model):
```

Comment:
Model name should start with capital and should be singular

Reply:
DOnne